sdks/python/apache_beam/typehints/opcodes.py

```diff
@@ -327,7 +339,18 @@ def call_function(state, arg, has_var=False, has_kw=False):
 
 
 def make_function(state, arg):
-  state.stack[-arg - 1:] = [Any]  # a callable
+  """Creates a function with the arguments at the top of the stack.
+  """
+  # TODO(luke-zhu): Handle default argument types
```

Comment:
Lets make a JIRA issue for this to track it and so other interested folks can help out :)